### DISCUSSION TITLE
beforeRequest indicator with boost

### DISCUSSION THREAD

**Discussion by stloc** [bigskysoftware/htmx#2193] [Q&A]
Hello,

with hx-boost="true" on body I print a custom loader on link click
 
  document.body.addEventListener('htmx:beforeRequest', function() {
        loader();
    });

but loader keep on screen when you go back with the browser

a solution please ?

**Comment by mdesco-crgl** [comment]
why not use https://htmx.org/extensions/loading-states/ instead of beforeRequest
